crescent/commands/decorators.py

```diff
@@ -60,10 +60,12 @@ def command(
 @overload
 def command(
     *,
-    guild: Snowflakeish | None = None,
-    name: str | None = None,
-    description: str | None = None,
-    deprecated: bool = False,
+    guild: Snowflakeish | None = ...,
+    name: str | None = ...,
+    description: str | None = ...,
+    default_member_permissions: UndefinedType | int | Permissions = ...,
+    dm_enabled: bool = ...,
```

Comment:
IDK what discord names it, but unless this is the exact name used by discord, maybe use "allow_dm_usage"?

Reply:
`dm_enabled` is hikari's impl name. Discord's official name is `dm_permission`.